opentech/apply/funds/management/commands/migration_base.py

```diff
@@ -260,3 +283,17 @@ def get_workflow_state(self, node):
 
     def nl2br(self, value):
         return value.replace('\r\n', '<br>\n')
+
+    def process_file(self, value):
+        if isinstance(value, dict):
+            value = [value]
+
+        files = []
+
+        for file_data in value:
+            file_path = 'files/private/application_upload/' + file_data['filename']
```

Comment:
Application uploads are in a number of different directories unfortunately. The code need to look in file_data['uri'] and parse it for path and filename.

Reply:
Fixed in 050f1de